drake/bindings/matlab/+autodiffutils/toAutoDiff.m

```diff
@@ -0,0 +1,18 @@
+function obj = toAutoDiff(value, derivatives)
+
+if nargin < 2
+  derivatives = eye(numel(value));
+end
+
+if size(value, 2) == 1
+  if size(value, 1) <= 73
+    obj = autodiffutils.AutoDiffVectorMax73(value, derivatives);
```

Comment:
am i missing something?  how does this work?  I don't see that method in the autodiffutils folder?
also, is bindings/matlab definitely the right place for this?  it's too bad to add the directory to the path just for this one file.  perhaps it should live under util?